eras/allegra/impl/src/Cardano/Ledger/Allegra/TxBody.hs

```diff
@@ -218,8 +238,8 @@ deriving newtype instance
   NoThunks (AllegraTxBody era)
 
 deriving newtype instance
-  ( NFData (TxOut era)
-  , NFData (PParamsUpdate era)
+  ( NFData (PParamsUpdate era)
+  , NFData (TxOut era)
```

Comment:
alphabetical order?

Reply:
Do you mean it *should* be in an alphabetical order or are you saying that this edit is redundant? We should probably make fourmolu do something about the ordering of constraints.

Reply:
I think Jared is asking did you make this change with motivation to sort the constraints? Cause this change isn't really necessary and that could be the only motivating factor.

My personal guess it was accidental. Am I right? :slightly_smiling_face:

Reply:
Yeah, I already reverted it based on one of your comments.

Reply:
yep, I was just trying to understand the motivation. :) I'd love it if fourmolu would be opinionated about the constraint order.